src/site/xdoc/server/quick-start-cassandra.xml

```diff
@@ -183,6 +190,123 @@ Step 9: Monitor
 
 </source>
 
+  <p>Second method, with docker-compose:</p>
+  
+<source>
+Step 0: Requirements
+####################
+
+  * 2GB RAM
+  * Docker 1.7.1+
+  * wget
+
+Step 1: Download
+#################
+
+  * Get the James docker-compose file
+
+<i>$ wget https://raw.githubusercontent.com/apache/james-project/master/dockerfiles/run/docker-compose.yml</i>
+
+Step 2: Start
+#############
+
+  * Run James
+
+<i>$ docker-compose up</i>
+
+Step 3: Create Domains and Users
+################################
+
+Time to add domains and users.
+
+<i>$ docker exec james java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999 adddomain <b>DOMAIN</b></i>
+
+<i>$ docker exec james java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999  adduser <b>USER_MAIL_ADDRESS</b> <b>PASSWORD</b></i>
+
+Where :
+- <b>DOMAIN</b> is the domain you want to handle with this server
+- <b>USER_MAIL_ADDRESS</b> user's email
+- <b>PASSWORD</b> user's password
+
+
+Step 4: Test
+############
+
+$ telnet <b>HOSTNAME</b> 25
+Trying <b>HOSTNAME</b>...
+Connected to <b>HOSTNAME</b>.
+Escape character is '^]'.
+220 172.16.1.131 SMTP Server (JAMES SMTP Server 3.0-beta4) ready Sat, 6 Nov 2010 17:31:33 +0100 (CET)
```

Comment:
Still beta4 for advertise version ? :-)

You performed these commands in 2010?